.github/workflows/continuous-delivery.yml

```diff
@@ -487,6 +498,71 @@ jobs:
           # TAGS could be separated by newlines or commas
           image=$(sed -n '1{s/,.*//; p}' <<< "$TAGS")
           echo "::set-output name=image::${image}"
+      -
+        name: Generate manifest for operator deployment
+        id: generate-manifest
+        run: |
+          make generate-manifest
+      -
+        name: Upload the operator manifest as artifact in workflow
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: operator-manifest.yaml
+          path: dist/operator-manifest.yaml
+          retention-days: 7
+      -
+        name: Replicate the artifact or not (to cloudnative-pg/artifacts repo)
```

Comment:
I do not like calling this step "replicate". It is publishing. I would change the step title and all the related variables.